source/beamline/BioXAS/BioXASSSRLMonochromatorEnergyControl.cpp

```diff
@@ -0,0 +1,23 @@
+#include "BioXASSSRLMonochromatorEnergyControl.h"
+
+BioXASSSRLMonochromatorEnergyControl::BioXASSSRLMonochromatorEnergyControl(const QString &name, const QString &readPVname, const QString &writePVname, const QString &movingPVname, const QString &stopPVname, QObject *parent, double tolerance, double moveStartTimeoutSeconds, AMAbstractControlStatusChecker *statusChecker, int stopValue, const QString &description) :
+    AMPVwStatusControl(name, readPVname, writePVname, movingPVname, stopPVname, parent, tolerance, moveStartTimeoutSeconds, statusChecker, stopValue, description)
+{
+
+}
+
+BioXASSSRLMonochromatorEnergyControl::~BioXASSSRLMonochromatorEnergyControl()
+{
+
+}
+
+AMControl::FailureExplanation BioXASSSRLMonochromatorEnergyControl::move(double setpoint)
```

Comment:
Is this to compensate for the fact that 0.001 is the precision of the mono energy PV and if you send an identical move it doesn't do anything - leading to AM procedures hanging?